README.md

```diff
@@ -15,7 +15,7 @@ use npm.
 
 ### Default Installs
 
-Corepack is distributed by default with Node.js 14.19.0 and 16.9.0, but is
+Corepack is distributed by default with all recent Node.js versions, but is
```

Comment:
I would prefer a specific version here because this is about the expectation of `corepack` cli being distributed with `node` in the current PATH.

Alternatively, we could link to the [official docs](https://nodejs.org/api/corepack.html) that mention the versions in the first section:

```
Added in: v16.9.0, v14.19.0
```